ctapipe/reco/sklearn.py

```diff
@@ -292,7 +295,7 @@ class SKLearnClassificationReconstructor(SKLearnReconstructor):
     ).tag(config=True)
 
     invalid_class = traits.Integer(
-        default_value=-1, help="The label to fill in case no prediction could be made"
+        default_value=-1, help="The label to fill in case no prediction could be made."
```

Comment:
To me "label" sounds very string-like, might be better to say "value" when talking about an integer.

Reply:
How about "label value" like for `positive_class`?

Reply:
> To me "label" sounds very string-like

This is the "label" in the machine learning sense: i.e. the true / target quantity.

Reply:
Label value sounds fine @LukasBeiske